Application Lifecycle Management/instances-api.md

```diff
@@ -333,85 +390,170 @@ Creates a new instance and gets the created instance.
 | name | string | Instance name. |
 | instance_tags | array | List of tags defined at deployment time. |
 
-```
-Headers:
+### Response
+#### Normal Response Codes
+- **202** OK
 
-Content-Type: application/json
-Authorization: Bearer your_json_web_token
-ElasticBox-Release: 4.0
-```
+#### Common Error Response Codes
+- **400** Invalid Data
 
-```
-Body: notice that the request schedules the instance to shut down in five hours.
-{
-   "schema": "http://elasticbox.net/schemas/deploy-instance-request",
-   "owner": "operations",
-   "name": "ScriptBoxSample",
-   "box": {
-       "id": "7a99f75c-30e6-4986-9059-f6889d1ff5f9",
-       "variables": [{
-           "name": "variable_name",
-           "type": "Text",
-           "value": "overridden variable value at deployment time",
-           "required": true,
-           "visibility": "public"
-       }]
-   },
-   "instance_tags": [
 
-   ],
-   "automatic_updates": "off",
-   "policy_box": {
-       "id": "e57466ee-7094-4bd4-9121-a6df4395d493",
-       "variables": [
+#### Response Body
```

Comment:
Add here next lines with section Response Parameters
#### Response Parameters
See description in previous command